GiniVision/Classes/Core/GiniError.swift

```diff
@@ -46,7 +56,30 @@ public protocol GiniVisionError: Error {}
     
     /// Camera roll can not be loaded because the user has denied authorization in the past.
     case photoLibraryAccessDenied
+    
+    /// Number of files picked exceeded
```

Comment:
Exceeded what?
"/// Max Number of files picked exceeded"

That also applies for the variable name. It could be more descriptive. For instance, `maxFilesCountExceeded `